ci/operator_build_release.sh

```diff
@@ -53,9 +56,14 @@ if [[ $release_tag == *"-"* ]]; then
   prev_tag=$(echo "$tags" | sed -n '1 p')
 fi
 
-# Push operator image.
-bazel run --stamp -c opt --//k8s:image_version="${release_tag}" \
-    --stamp "${build_type}" //k8s/operator:operator_images_push
+push_images_for_arch "x86_64" "//k8s/operator:operator_images_push" "${release_tag}" "${build_type}" "${extra_bazel_args[@]}"
+push_images_for_arch "aarch64" "//k8s/operator:operator_images_push" "${release_tag}" "${build_type}" "${extra_bazel_args[@]}"
+
+while read -r image;
```

Comment:
maybe we could extract this into `image_utils.sh` since we need to do it in both places